libkbfs/context.go

```diff
@@ -4,19 +4,7 @@
 
 package libkbfs
 
-import (
-	"net"
-
-	"github.com/keybase/client/go/libkb"
-	"github.com/keybase/go-framed-msgpack-rpc/rpc"
-)
+import "github.com/keybase/kbfs/env"
 
 // Context defines the environment for this package
-type Context interface {
-	GetRunMode() libkb.RunMode
-	GetLogDir() string
-	GetDataDir() string
-	ConfigureSocketInfo() (err error)
-	GetSocket(clearError bool) (net.Conn, rpc.Transporter, bool, error)
-	NewRPCLogFactory() *libkb.RPCLogFactory
-}
+type Context env.Context
```

Comment:
Is there a reason this still exists? Perhaps we can just replace `libkbfs.Context` with `env.Context` everywhere?

Reply:
I was trying to minimize code changes. And it's harmless to have it there for now.

Reply:
Sounds good.